# HG changeset patch
# User Julien Grall <julien.grall@linaro.org>
# Date 1381829069 -3600
# Node ID 3d8219a6c2cd6f58297956aa5719d7fad472533b
# Parent  ef254252f52fa208468f225e765a0a3ad8270f6b
xen/evtchn: Fix build on ARM

The recent event channel changes introduced by commit a77eb86 and before...
break the compilation on Xen ARM. This commit adds missing includes in
common/event_fifo.c and include/xen/sched.h.

Signed-off-by: Julien Grall <julien.grall@linaro.org>
Acked-by: Ian Campbell <ian.campbell@citrix.com>

diff -r ef254252f52f -r 3d8219a6c2cd xen/common/event_fifo.c
--- a/xen/common/event_fifo.c	Mon Oct 14 17:46:18 2013 +0100
+++ b/xen/common/event_fifo.c	Tue Oct 15 10:24:29 2013 +0100
@@ -16,6 +16,7 @@
 #include <xen/event_fifo.h>
 #include <xen/paging.h>
 #include <xen/mm.h>
+#include <xen/domain_page.h>
 
 #include <public/event_channel.h>
 
diff -r ef254252f52f -r 3d8219a6c2cd xen/include/xen/sched.h
--- a/xen/include/xen/sched.h	Mon Oct 14 17:46:18 2013 +0100
+++ b/xen/include/xen/sched.h	Tue Oct 15 10:24:29 2013 +0100
@@ -24,6 +24,7 @@
 #include <public/sysctl.h>
 #include <public/vcpu.h>
 #include <public/mem_event.h>
+#include <public/event_channel.h>
 
 #ifdef CONFIG_COMPAT
 #include <compat/vcpu.h>
